correct an error description of Request instance behavior (#27697)
mdn/content: files/en-us/web/api/request/index.md

- {{domxref("Request.text()")}}
  - : Returns a promise that resolves with a text representation of the request body.

> > **Note:** The request body functions can be run only once; subsequent calls will raise an error showing that the body stream has already used.

Author: ```suggestion
> **Note:** The request body functions can be run only once; subsequent calls will reject with TypeError showing that the body stream has already used.
```
